tspng/implantation.py

```diff
@@ -0,0 +1,82 @@
+# import statements
+import os
+import json
+
+from PIL import Image
+from PIL.PngImagePlugin import PngInfo
+from tspng import MIME_TYPE
+from typing import Dict
+
+
+def _implant_data(data: str, image: str, mime_type: str = MIME_TYPE) -> Dict:
```

Comment:
To make this more easily testable, a `dst` argument, short for `destination`, should be added and implemented. It would be `Optional[str]` type and default to `None`. If the value is `None`, then the image file is created in the same location as the `image` file. If the value is a path as a string to a directory that already exists, then the `.ts.png` image will be saved to the directory with the file stem being the same as the `image` file. If the value is a path as a string and it does not exist, then the assumption is that it should be the output file name.

With this added, the [`tmp_path`](https://docs.pytest.org/en/6.2.x/tmpdir.html) fixture can be used in tests without having to worry about cleaning up after tests.